.gitignore

```diff
@@ -6,12 +6,16 @@
 
 .idea/misc.xml
 
+.idea/preferred-vcs.xml
```

Comment:
Are you certain this WebStorm project file should be ignored? What does it contain?